modules/lb_internal/variables.tf

```diff
@@ -42,8 +42,19 @@ variable "subnetwork" {
   type = string
 }
 
+variable "ip_version" {
```

Comment:
shouldn't this one be `IPV4` as the default ?

Reply:
`ip_version` is a new parameter and it was assumed as non-existent (null) in the resources that were previously created using previous module versions. 
Because changing `ip_version`causes LB to be recreated, to provide less disruptive module upgrade experience I'm using `null` as default value.